ethereum/contracts/IncentivizedOutboundChannel.sol

```diff
@@ -19,6 +30,16 @@ contract IncentivizedOutboundChannel is OutboundChannel {
         override
     {
         nonce = nonce + 1;
-        emit Message(msg.sender, nonce, payload);
+        emit Message(msg.sender, nonce, payload, msg.value);
+    }
+
+    modifier onlyFeeController {
+        require(msg.sender == feeController);
+        _;
+    }
+
+    function setGasFee(uint256 _fee) public onlyFeeController {
```

Comment:
Unless `feeController` is set in a contract that extends `IncentivizedOutboundChannel` the modifier will reject all attempts to update the gas fee.